src/fragments/lib/push-notifications/common/existing-resources.mdx

```diff
@@ -0,0 +1,18 @@
+Existing Amazon Pinpoint resources can be used with the Amplify Libraries by referencing your **Application ID** and **Region** in your `amplifyconfiguration.json` file.
+
+```json
+"notifications": {
+    "plugins": {
+        "awsPinpointPushNotificationsPlugin": {
+            "appId": "[APP ID]",
+            "region": "[REGION]"
+        }
+    }
+}
+```
+
+- **pinpoint**
```

Comment:
The `"pinpoint"` object no longer exist in the final configuration structure, so remove it from here

Reply:
good catch, removed.